compile.sh

```diff
@@ -31,6 +31,12 @@ FORCE_CHECKOUT="yes"			# ignore manual changes to source
 BUILD_ALL="no"				# cycle through available boards and make images or kernel/u-boot packages.
 					# set KERNEL_ONLY to "yes" or "no" to build all packages/all images
 
+PRIVATE_CCACHE="no"			# Use separate ccache directory located in build tree ($DEST/ccache)
+					# Can be used to avoid problems with ownershipt on $HOME/.ccache. 
+					# Set to "yes" if you are using build script as non-root user with "sudo"
+					# note: to use a ramdisk for ccache, create first a symlink to $DEST/ccache
+                                        # Set to "no" check if $DEST/ccache exists and delete (directory/symlink)
```

Comment:
OK, I understand the "need" for that cleanup, even if a bit unfortunate...
But untangling the need for running as "root" would be a bigger task, so I retract my objection.
LGTM with the other small glitches fixed...

PS: I'm also french (from Alsace)